modules/assim.batch/R/pda.emulator.R

```diff
@@ -211,15 +230,17 @@ pda.emulator <- function(settings, params.id = NULL, param.names = NULL, prior.i
       for (i in seq_len(settings$assim.batch$n.knot)) {
         align.return <- pda.get.model.output(settings, run.ids[i], bety, inputs)
         model.out[[i]] <- align.return$model.out
-        if(!is.na(model.out[[i]])){
+        suppressWarnings(if(!is.na(model.out[[i]])){
           inputs <- align.return$inputs
-        } 
+        })
```

Comment:
here model.outputs[[i]] itself is a list, and all that sublists are data frames of aligned model outputs

I want to check if the model run failed or not, so just looking at the first index is enough, the check gives me "using the first element" sorts of warnings, I was just being lazy :) I'll try to specify the check more precisely